src/sentry/utils/celery.py

```diff
@@ -1,9 +1,62 @@
+from collections.abc import Mapping, MutableSequence, Sequence
 from random import randint
 from typing import Any
 
 from celery.schedules import crontab
+from kombu import Queue
+
+from sentry.conf.types.celery import SplitQueueSize
 
 
 def crontab_with_minute_jitter(*args: Any, **kwargs: Any) -> crontab:
     kwargs["minute"] = randint(0, 59)
     return crontab(*args, **kwargs)
+
+
+def build_queue_names(base_name: str, quantity: int) -> Sequence[str]:
+    ret = []
+    for index in range(quantity):
+        name = f"{base_name}_{index + 1}"
+        ret.append(name)
+    return ret
+
+
+def make_split_queues(config: Mapping[str, SplitQueueSize]) -> Sequence[Queue]:
```

Comment:
We don't seem to call this outside of tests. Do we need to setup the post_process_transactions split queues, or are we going to rely on the patching being done in getsentry/ops for a while longer?

Reply:
There was a debate in the previous PR whether we should add the queues in sentry or getsentry.
https://github.com/getsentry/sentry/pull/76494#discussion_r1742801927.
What do you think? 

I am a bit biased towards adding them in the sentry configuration rather than getsentry to create fewer surprises.
Either way, in order to rollout safely, I need to deploy this first, then adapt the current patch to avoid queue duplication (if you declare the same queue twice two queues actually show up), then I can switch to using this method.